tests/common/scripts/merging/test_merge_instant.py

```diff
@@ -0,0 +1,16 @@
+from common.scripts.merging import merge_instant
+import pytest
+
+
+def test_merge_instant():
+    assert merge_instant("2015-02-07", "13:28:17") == "2015-02-07T13:28:17+02:00"
+
+    with pytest.raises(ValueError) as exc_info:
+        merge_instant("11111", "2222", "3333")
+
+    assert exc_info.type is ValueError
```

Comment:
nit: you don't need this because `pytest.raises(ValueError)` already checks that the exception is of type ValueError